docs/resources/grant_privileges_to_account_role.md

```diff
@@ -0,0 +1,396 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "snowflake_grant_privileges_to_account_role Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  
+---
+
+~> **Note** This is a preview resource. It's ready for general use. In case of any errors, please file an issue in our GitHub repository.
+
+
+!> **Warning** Be careful when using `always_apply` field. It will always produce a plan (even when no changes were made) and can be harmful in some setups. For more details why we decided to introduce it to go our document explaining those design decisions (coming soon).
+
+# snowflake_grant_privileges_to_account_role (Resource)
+
+
+
+## Example Usage
+
+```terraform
+resource "snowflake_database" "db" {
+  name = "database"
+}
+
+resource "snowflake_role" "db_role" {
+  name = "role_name"
+}
+
+##################################
+### on account privileges
+##################################
+
+# list of privileges
+resource "snowflake_grant_privileges_to_account_role" "example" {
+  privileges = ["CREATE DATABASE", "CREATE USER"]
+  role_name  = snowflake_role.db_role.name
```

Comment:
We changed role_name to account_role_name. Please change the examples.

Reply:
DOne